sys/net/gnrc/network_layer/ndp/gnrc_ndp.c

```diff
@@ -28,11 +28,13 @@
 
 #include "timex.h"
 
-#define ENABLE_DEBUG    (0)
+#define ENABLE_DEBUG 0
 #include "debug.h"
 
+#if ENABLE_DEBUG
 /* For PRIu8 etc. */
 #include <inttypes.h>
+#endif
```

Comment:
I think this will result in issues later on. When using `PRIu8` and friends in `DEBUG()` statements, those have to be present - even if the `printf()` call that gets `PRIu8` passed as argument get eliminated during optimization.

Reply:
Fixed that one.